src/test/kotlin/com/jetbrains/snakecharm/SmkStatementMoverTest.kt

```diff
@@ -0,0 +1,253 @@
+package com.jetbrains.snakecharm
+
+import com.intellij.openapi.actionSystem.IdeActions
+import com.intellij.openapi.fileEditor.FileDocumentManager
+import com.jetbrains.python.psi.LanguageLevel
+import features.glue.SnakemakeWorld.myFixture
+
+class SmkStatementMoverTest: SnakemakeTestCase() {
+    private fun doTest() {
+        val testName: String = getTestName(true)
+
+        myFixture?.configureByFile("mover/$testName.smk")
```

Comment:
Let's rewrite this test case using cucumber. See `line_mover.feature` example

Reply:
Done